### PR TITLE
Added Toasts + settings for it, autoTool boolean, fixed click gui info …

### PR DESCRIPTION
**Pull request by RealIndrit**
…message

### CODE REVIEW

File: src/api/java/baritone/api/utils/Helper.java [hunk 1]
```diff
@@ -60,6 +62,43 @@ static ITextComponent getPrefix() {
         return prefix;
     }
 
+    /**
+     * Send a message to the toaster to show it as a popup
+     *
+     * @param title The title to display in popup as textcomponent
+     * @param message The message to display in popup as textcomponent
+     */
+    default void logToast(ITextComponent title, ITextComponent message){
```
Comment (StijnSimons) [inline src/api/java/baritone/api/utils/Helper.java]: default methods XDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD

Comment (RealIndrit) [reply src/api/java/baritone/api/utils/Helper.java]: Well, how would you do it? My idea was to basically to keep all "interaction response stuff" in the same file, the "Helper" interface...

**Comment by leijurv** [comment]
???? we dev on master and merge upward into 1.13.2 and 1.14.4 i won't merge a new feature onto 1.13.2 only??

**Comment by RealIndrit** [comment]
Aight ill make a pull request later on 1.12.2, currently remkaing farm process caus its kinda retarded currently tbh

**Pull request closed without merging**